src/lib/core/CHIPConfig.h

```diff
@@ -2108,6 +2108,27 @@
 #define CHIP_CONFIG_PERSISTED_STORAGE_KEY_TYPE const char *
 #endif
 
+/**
+ * @def CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_KEY
+ *
+ * @brief
+ *   Lifetime Counter Key.
+ */
+#ifndef CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_KEY
+#define CHIP_CONFIG_LIFETIIME_PERSISTED_COUNTER_KEY "rotation-counter"
```

Comment:
if the constant is 'LIFETIME COUNTER' should the key also be 'lifetime-counter' ?

Reply:
fixed